index.html

```diff
@@ -885,17 +885,15 @@ <h2>
         Privacy and Security Considerations
       </h2>
 
+     
       <section>
-        <h2>Access to sensors on a user’s device</h2>
+        <h3>Access to aspects of a user’s local computing environment</h3>
         <p>
-          This specification does not allow an origin access to sensors on a user’s device directly, but information exposed by sensors (e.g., the screen orientation angle) can be accessed indirectly.
-        </p>
-      </section>
-
-      <section>
-        <h2>Access to aspects of a user’s local computing environment</h2>
-        <p>
-          The screen orientation type and angle of the device can be accessed with the API specified in this document, and can be a potential fingerprinting vector.
+          The screen orientation type and angle of the device can be accessed with the API specified in this document,
+          and can be a potential fingerprinting vector. However using this information is very difficult compared to other fingerprinting information. 
+          Aggregating information derived from this API is not very helpful in fingerprinting. 
+          In particular the Device Orientation API provides a far more detailed version of the same information. 
```

Comment:
By "Device Orientation API", do you mean the [DeviceOrientation Event Specification](https://w3c.github.io/deviceorientation/spec-source-orientation.html), or the [motion sensor specs](https://w3c.github.io/motion-sensors/)?

Reply:
I meant the event spec, but actually it is true of both.